pkg/metrics/metrics.jsx

```diff
@@ -592,7 +644,18 @@ class CurrentMetrics extends React.Component {
         return (
             <Gallery className="current-metrics" hasGutter>
                 <Card id="current-metrics-card-cpu">
-                    <CardTitle>{ _("CPU") }</CardTitle>
+                    <CardHeader className='align-baseline'>
+                        <CardTitle>{ _("CPU") }</CardTitle>
+                        { !isNaN(this.cpuTemperature) &&
+                        <Flex className="temperature-flex" spaceItems={{ default: 'spaceItemsXs' }}>
```

Comment:
Using a flex here feels overkill - an `&nbsp` between icon and temperature feels enough? Moreover, we really don't want to break up the icon and the temp, as then you would have some "detached" warning icon floating around somewhere.

Reply:
Changed to `&nbsp;` which is more correct IMHO. Renamed "temperature-flex" to just "temperature", we only use that class in the test and it should convey the meaning, not the implementation.